cmd/otelcol/Dockerfile

```diff
@@ -6,6 +6,10 @@ COPY otelcol /
 # Note that this shouldn't be necessary, but in some cases the file seems to be
 # copied with the execute bit lost (see #1317)
 RUN chmod 755 /otelcol
```

Comment:
750?

Reply:
This is what it was.  IF you think the chmod of the default docker file needs to change, then perhaps that deserves a separate bug/discussion.